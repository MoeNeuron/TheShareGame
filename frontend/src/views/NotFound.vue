<template>
  <div>
    Page could not be found! Are your sure the url is correct?
  </div>
</template>

<script>
export default {
  name: 'NotFound',
};
</script>

<style scoped>

</style>
