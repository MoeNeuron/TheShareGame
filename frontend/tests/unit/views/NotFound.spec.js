import { mount } from '@vue/test-utils';
import NotFound from '@/views/NotFound.vue';

describe('Welcome.vue', () => {
  it('matches Snapshot', () => {
    expect(mount(NotFound)).toMatchSnapshot();
  });
});
