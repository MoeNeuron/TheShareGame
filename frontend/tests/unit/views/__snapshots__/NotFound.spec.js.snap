// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome.vue matches Snapshot 1`] = `
<div>
  Page could not be found! Are your sure the url is correct?
</div>
`;
